package.json

```diff
@@ -48,14 +59,31 @@
   "engines": {
     "node": ">=12.14.0"
   },
-  "dependencies": {},
+  "dependencies": {
+    "dotenv": "^8.2.0",
+    "express": "^4.17.1",
+    "express-handlebars": "^4.0.4",
+    "handlebars": "^4.7.6",
+    "shelljs": "^0.8.4"
+  },
```

Comment:
Playground only needs to work for development, not in a shipped package. Therefore, these are all development dependencies. If someone wants to deploy the playground they can always do `npm install --only=dev`

Reply:
ok.